### ISSUE TITLE
Nuxt webpack error in browser only, not during build

### ISSUE THREAD

**Issue by pimlie** [nuxt/nuxt.js#1109]
This looks like something with my project specifically (a new empty nuxt project does work), but I am a bit lost and after looking for 2 days I dont know anymore. Hopefully somebody here can put me on the right track.

The issue is that building my project works fine, e.g. after `npm run dev` I have the following output  (using vue version 2.3.4):
```
DONE  Compiled successfully in 463ms
> Open http://0.0.0.0:3000
nuxt:render Rendering url / +6s
```

But in my browser I receive the following error:
```
Nuxt.js Error:

TypeError: __webpack_require__.i(...) is not a function
    at Object.module.exports.Object.defineProperty.value (~/babel-runtime/helpers/typeof.js:13:63)
    at __webpack_require__ (webpack:/webpack/bootstrap 32d2034bb6fb3978fa81:25:0)
    at Object.<anonymous> (server-bundle.js:480:122)
    at __webpack_require__ (webpack:/webpack/bootstrap 32d2034bb6fb3978fa81:25:0)
    at Object.module.exports.__webpack_exports__.a (server-bundle.js:1290:66)
    at __webpack_require__ (webpack:/webpack/bootstrap 32d2034bb6fb3978fa81:25:0)
    at Object.<anonymous> (server-bundle.js:1881:65)
    at __webpack_require__ (webpack:/webpack/bootstrap 32d2034bb6fb3978fa81:25:0)
    at server-bundle.js:95:18
    at Object.<anonymous> (server-bundle.js:98:10)
    at evaluateModule (/var/www/nuxt/node_modules/vue-server-renderer/build.js:5820:21)
    at /var/www/nuxt/node_modules/vue-server-renderer/build.js:5878:18
    at Promise (<anonymous>)
    at /var/www/nuxt/node_modules/vue-server-renderer/build.js:5870:14
    at Nuxt.renderToString (/var/www/nuxt/node_modules/vue-server-renderer/build.js:6022:9)
    at P (/var/www/nuxt/node_modules/nuxt/node_modules/pify/index.js:49:6)
```

I have removed node_modules already and re-installed all dependencies from scratch but that doesnt help. Any ideas where I should be looking?

**Comment by pimlie** [comment]
It seems a couple of things were at play here. First dependencies were not correctly installed for some reason, e.g. running npm install or yarn resulted in different packages / package versions. After restructuring the dependencies in my package.json and removing some explicit version definitions I finally managed to have a list without any unmet dependencies. The only thing I did was upgrading to Node v8 this weekend, but currently it works both with Node v7 as v8.

Now the culprit for the webpack error seems to be the solution from #924. If I comment out the babelloader.exclude lines I dont get the Nuxt webpack error. I am not sure why I get that error now, because when I implemented those lines I know for sure it worked. Will try to get a repro working for this.
